exercises/largest-series-product/src/example.c

```diff
@@ -14,7 +14,7 @@ static bool is_digit(char c)
    return (c >= '0') && (c <= '9');
 }
 
-int64_t largest_series_product(char *digits, size_t span)
+int64_t largest_product(char *digits, size_t span)
```

Comment:
I know that this technically follows the specification... but why? The exercise is called `largest-series-product`, so having a function called `largest_product` instead of `largest_series_product` just seems needlessly inconsistent.